consensus/polybft/state_stats.go

```diff
@@ -154,3 +158,44 @@ func (s *State) startStatsReleasing() {
 		prev = stats
 	}
 }
+
+func polybftMetrics(rootnodeURL string,
```

Comment:
WDYT about renaming this to `publishRootchainMetrics` and publishing it from `checkpoint_manager`? The reasoning is that checkpointManager already contains rootchainTxRelayer (that exposes `jsonrpc.Client`), which can be reused. Then, I would also consider moving the following metrics there as well (currently, only checkpoint proposers are publishing it but on checkpoint blocks only, but this way all validators would publish it on a more regular basis):

`metrics.SetGauge([]string{"bridge", "checkpoint_block_number"}, float32(header.Number))`